core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -156,9 +157,15 @@ private static void init(String sitePropertiesFile, boolean includeSystemPropert
           String.format(message, Constants.WORKER_RPC_PORT));
       Preconditions.checkState(System.getProperty(Constants.WORKER_WEB_PORT) == null,
           String.format(message, Constants.WORKER_WEB_PORT));
-      PROPERTIES.setProperty(Constants.WORKER_DATA_PORT, "0");
-      PROPERTIES.setProperty(Constants.WORKER_RPC_PORT, "0");
-      PROPERTIES.setProperty(Constants.WORKER_WEB_PORT, "0");
+      PROPERTIES_MAP.put(Constants.WORKER_DATA_PORT, "0");
+      PROPERTIES_MAP.put(Constants.WORKER_RPC_PORT, "0");
+      PROPERTIES_MAP.put(Constants.WORKER_WEB_PORT, "0");
+    }
+  }
+
+  private static void putFromProperties(Map<String, String> map, Properties properties) {
```

Comment:
it looks to me that all references to `putFromProperties` make `PROPERTIES_MAP` as the first argument. In this case, do we still need make it a parameter?